scripts/quickstart.sh

```diff
@@ -262,6 +286,16 @@ for i in $(seq 0 1); do
     --tracing.config="${QUERIER_JAEGER_CONFIG}" \
     --query.replica-label receive_replica \
     ${STORES} &
+
+  sleep 0.5
+
+  ${THANOS_EXECUTABLE} query-frontend \
```

Comment:
This is not how it suppose to work, unfortunately. Those replicas have to be separate for better horizontal scalability. This means there should be a load balancing in between - ideally round robin load balancer for better distribution.

We can add some nginx for it, but ideally, we could work towards this: https://github.com/thanos-io/thanos/issues/3373

Let me know if this makes sense :hugs: